gwswlib/sql_models/constants.py

```diff
@@ -0,0 +1,226 @@
+def reversed_dict(d):
+    """Create a reverse lookup dictionary"""
+    return dict([(b, a) for a, b in d])
+
+
+def choices_as_set(d):
+    """Handy to check if domain is correct"""
```

Comment:
Eerste regel moet zijn wat de functie doet/teruggeeft.

Ook hier weer even zeggen wat `d` is.